### PR TITLE
Adds two new lever action rifles.

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request
Adds the Absolution Lever Action Rifle, chambered in 357, and Conflagration Lever Action Shotgun. Pictured below.
imagen
imagen
imagen

Spriting by RyeRice, Names by Valorium, Descriptions by Apogee, and coding by me!!

## Why It's Good For The Game
Upgrades to the .38 lever action rifle that provide a nice variety, with the lever-action shotgun being an inbetween of a double-barrel and pump action shotgun and the 357 being more expensive and with a smaller ammo capacity but with a lot more power.
## Changelog

:cl:
add: To-do once i get the proper names and all
/:cl:

### CODE REVIEW

**Review by rye-rice — APPROVED** [review]
Made the sprites for this, though they aren't implmented yet

**Comment by Martinpachu** [comment]
This Pull Request is complete. Please review ! It is so late in the night

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -692,6 +692,70 @@ EMPTY_GUN_HELPER(shotgun/bulldog/inteq)
...
+	spawnwithmagazine = FALSE
+
+/obj/item/gun/ballistic/shotgun/flamingarrow/conflagration
+	name = "HP Conflagration"
+	base_icon_state = "conflagration"
+	icon_state = "conflagration"
+	item_state = "conflagration"
+	fire_sound = 'sound/weapons/gun/shotgun/shot.ogg'
+	desc = "A lightweight lever-action shotgun with a 5 round ammunition capacity. The lever action allows it to be cycled quickly and acurrately. In theory, you could ever operate it one-handed. Chambered in 12g."
+	sawn_desc = "A lever action shotgun that's been sawed down for portability. The recoil makes it mostly useless outside of point-blank range, but it hits hard for it's size and, more importantly, can be flipped around stylishly."
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]:
```suggestion
	sawn_desc = "A lever action shotgun that's been sawed down for portability. The recoil makes it mostly useless outside of point-blank range, but it hits hard for its size and, more importantly, can be flipped around stylishly."
```

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 2]
```diff
@@ -692,6 +692,70 @@ EMPTY_GUN_HELPER(shotgun/bulldog/inteq)
 		item_state = "flamingbolt_sawn"
 		mob_overlay_state = item_state
 
+/obj/item/gun/ballistic/shotgun/flamingarrow/absolution
+	name = "HP Absolution"
+	base_icon_state = "absolution"
+	icon_state = "absolution"
+	item_state = "absolution"
+	fire_sound = 'sound/weapons/gun/revolver/shot.ogg'
+	desc = "A large lever-action rifle with hand-stamped Hunter's Pride marks on the receiver and an 8 round ammunition capacity. More powerful than the Flaming Arrow, the Absolution is a popular pick for hunting larger fauna like bears and goliaths, especially whena bolt action's slower rate of fire would be a liability. Chambered in .357."
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]:
```suggestion
	desc = "A large lever-action rifle with hand-stamped Hunter's Pride marks on the receiver and an 8 round ammunition capacity. More powerful than the Flaming Arrow, the Absolution is a popular pick for hunting larger fauna like bears and goliaths, especially when a bolt action's slower rate of fire would be a liability. Chambered in .357."
```

**Comment by rye-rice** [comment]
During testing it seems to have gotten mixed reactions, however it had no issues i could see so into the queue it goes

**Pull request merged**